moments.yaml

```diff
@@ -0,0 +1,32 @@
+- id: WNP_MOMENTS_9
+  groups: [moments-pages]
+  content:
+    action:
+      data:
+        # Valid until November 29th EOD
+        expire: 1638144000000
+        # URL will redirect to correct locale based on browser settings
+        url: https://www.mozilla.org/firefox/welcome/9
+      id: moments-wnp
+    bucket_id: WNP_MOMENTS_9
+  targeting: localeLanguageCode in ["fr", "de", "es", "nl", "it"] &&
```

Comment:
It looks like only `fr` and `de` work with `/9` while `/10` supports `en-US`, `es-ES`, `nl` and `it`. (Not sure yet if `en-GB` or `es-MX` will redirect… I'll try it out now)

Reply:
Ah yup, `localeLanguageCode` will include all `es-*` including `es-MX` which would then redirect `/9` to the `de` version and `/10` to `en-US` (because there's no exact `es-MX` match, so the initial/default one gets shown). So I think we need 3 records?
1. /9 for de and fr
2. /10 for en, nl, it
3. es-ES/firefox/welcome/10 for es